mmv1/third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -128,11 +152,27 @@ resource "google_logging_metric" "logging_metric" {
 `, suffix, filter)
 }
 
+func testAccLoggingMetric_loggingBucket(suffix string, filter string, project_id string) string {
+	return fmt.Sprintf(`
+resource "google_logging_metric" "logging_metric" {
+  name        = "my-custom-metric-%s"
+  bucket_name = "projects/%s/locations/global/buckets/_Default"
+  filter      = "%s"
+
+  metric_descriptor {
```

Comment:
I'm forced to add this but doesn't look the same user experience that other UIs provide.

metric_descriptor should not be mandatory (see here: https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.metrics#LogMetric).

Anyway, APIs seem to auto-compute the values I added, if the value is not specified. We should put as optional/computed, but asking for a second opinion before updating more stuff in this patch.

Reply:
TL;DR imho we should add
required: false
default_from_api: true
in terraform.yaml

Reply:
Sounds good. Thanks.